src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -15,32 +15,33 @@
         </cd-table-actions>
 
         <div class="btn-group"
-             dropdown>
+             dropdown
+             *ngIf="advancedTableActions.length > 0">
```

Comment:
> Why not use cd-table-actions instead? To have an equal look and feel - as this drop down doesn't show the primary action in itself, but the cd-table-actions do.

We can't use the cd-table-actions component here because they relate to a table selection, but the options provided by the menu are independent.
It would be great if we improve the cd-table-actions component to deal with such menu options as well. However, I would suggest to do this in a separate issue (as your diff already contains a lot of changes that are not directly related to this issue).

Reply:
Please check out the diff. I submitted it into [this](https://github.com/Devp00l/ceph/commits/review-osd-config-option) branch. The `cd-table-actions` are working fine with the mentioned changes ;) That said, you can use it :D

Reply:
Thank you! Nevertheless I think the changes to the cd-table-actions component are worth their own PR.

Reply:
As you can see in the diff it will prevent some code you would add if not adapting the component to be used here. Also it will align the right action style and not use a different one, as the main action is always visible in the drop down, too, except in this custom action drop down.

Reply:
I still don't see the point why it can't be done by a follow-up PR.